definitions/infra-container/definition.yml

```diff
@@ -12,12 +12,14 @@ synthesis:
     - attribute: docker.containerId
 
   tags:
-    - container.state
-    - docker.state
-    - docker.containerId
-    - newrelic.integrationName
-    - newrelic.integrationVersion
-    - newrelic.agentVersion
+    - attribute: container.state
+      mutliValue: false
+    - attribute: docker.state
+      mutliValue: false
```

Comment:
I think these are solid use cases for it, but just want to highlight these lines

Reply:
s/mutliValue/multiValue